### ISSUE TITLE
Better name?

### ISSUE THREAD

**Issue by srmagura** [markerikson/react-community-tools-practices-cheatsheet#17]
It seems like the current name is either

* React Community Tools and Practices Cheatsheet (name of GitHub repo), or
* React Common Tools and Practices (what it says on the website)

I think having a short, memorable, and descriptive name will help the project succeed. Some ideas:

- The React Community Handbook _(I think this is my personal favorite)_
- The React Handbook
- The React Community Cheatsheet
- The React Compendium

"Common Tools and Practices in the React Ecosystem" could be a subtitle to provide additional detail.

**Comment by councit** [comment]
I am a fan of The React Community Handbook. Makes it feel inclusive and useful.

**Comment by AmyShackles** [comment]
I really like `The React Community Handbook` too!

She commented, belatedly.

**Comment by markerikson** [comment]
Yeah, I could get behind something like that.  Still feel like maybe throwing the word "Tools" in there - "React Community Tools Handbook"?
